pkg/git/git.go

```diff
@@ -218,6 +221,22 @@ func (r *repo) HasDiffs() (bool, error) {
 		errors.Wrapf(err, "error checking status of branch %q", r.currentBranch)
 }
 
+func (r *repo) GetDiffPaths() ([]string, error) {
+	resBytes, err := libExec.Exec(r.buildCommand("status", "-s"))
+	if err != nil {
+		return nil,
+			errors.Wrapf(err, "error checking status of branch %q", r.currentBranch)
+	}
+	diffs := strings.Split(string(resBytes), "\n")
+	diffs = diffs[:len(diffs)-1]
+	paths := make([]string, len(diffs))
+	for i, rawDiff := range diffs {
+		temp := strings.TrimSpace(rawDiff)
+		paths[i] = strings.SplitN(temp, " ", 2)[1]
+	}
+	return paths, nil
```

Comment:
nit: I think we may use bytes directly:

```suggestion
	paths := make([]string, 0)
	scanner := bufio.NewScanner(bytes.NewReader(resBytes))
	scanner.Split(bufio.ScanLines)
	for scanner.Scan() {
		temp := strings.TrimSpace(scanner.Text())
		if temp == "" {
			break
		}
		paths = append(paths, strings.SplitN(temp, " ", 2)[1])
	}
	return paths, nil
```

Reply:
I like this. I also think we don't need the check for breaking if temp == "". Verifying that first though.

Thanks for the suggestion.